discord/appinfo.py

```diff
@@ -107,6 +110,18 @@ class AppInfo:
     privacy_policy_url: Optional[:class:`str`]
         The application's privacy policy URL, if set.
 
+        .. versionadded:: 2.0
```

Comment:
Need new lines in between versionadded + the following attribute

Reply:
Not actually necessary

Reply:
Just to remain consistent with doc of other attributes, I've added them